manifests/init.pp

```diff
@@ -68,7 +68,7 @@
       $config_client_path_default = '/etc/default/openafs-client'
       $init_script_default        = '/etc/init.d/openafs-client'
       $init_template_default      = 'openafs-client-Ubuntu'
-      $package_name_default       = [ 'openafs-client', 'openafs-doc', 'openafs-krb5', 'openafs-modules-dkms', 'openafs-modules-source', ]
+      $package_name_default       = [ 'openafs-modules-dkms', 'openafs-modules-source', 'openafs-client', 'openafs-doc', 'openafs-krb5', 'dkms' ]
```

Comment:
just fyi: https://docs.puppetlabs.com/learning/ordering.html
If we need a special sequence here we need to code it. But I don't think it neccessary here.